### PR TITLE
Set up a "Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by Ibtisam34**
- [ ] By now I've used many different gems with Rails and this exercise will have me connect previous knowledge with new knowledge as I  get a chance to use Webpack with a bundling-rails gem.

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Ibtisam34,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive README. Also, you used the wrong workflow, you should request a merge to the develop branch, not main branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+ get '/api/random_greeting', to: 'api#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```
Comment (mukhammadrizooff) [inline config/routes.rb]: - [ ] [REQUIRED] Good job so far! There should be a main static route created:

* The `routes.rb` file has a route created (should be marked as `root` => `root#index`)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (mukhammadrizooff) [inline package.json]: - [ ] [REQUIRED] Good job so far! Your `package.json` file should need dependencies added:

* `react-dom`
* `babel-loader`

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @Ibtisam34  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔  Made suggested changes 👍  
✔ Descriptive PR 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**